crates/solidity/outputs/typescript/lib/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I suggest we remove that license file as well.

There are a bunch of other files like taplo/eslint which we need to remove/reconfigure as well, but I'm happy to take a stab at it as part of my upcoming work.

Reply:
k, I didn't want to remove things that might help your integration.

Reply:
And specifically, we might need to include that license because we use napi boilerplate.

Reply:
I will need to make sure that the produced `package.json` links to our original `LICENSE` file at the root of the repository. I just wanted to avoid having to licenses by accident. My understanding is that they are both the same one (MIT) with the same notice.

Reply:
The terms might be the same, but the copyright needs to be maintained.

Reply:
@AntonyBlakey  is the license of the template, or a license of NAPI itself?
https://github.com/napi-rs/package-template/blob/main/LICENSE

I would suggest avoiding changing the root license, at least until we figure out the NPM publishing process.

Reply:
cc @alcuadrado